spec/ttml1.xml

```diff
@@ -8094,7 +8110,8 @@ based timing or event based timing in <bibref ref="smil21"/>, where the specific
 <specref ref="time-expression-semantics-smpte"/> apply.</p>
 </item>
 </ulist>
-<p>The following example illustrates the timing semantics of anonymous spans that are children of sequential time containers. The words <emph>Hello</emph> and <emph>Allo</emph> are both contained in anonymous spans that are children of a sequential time container (the <code>p</code> element). As such, neither <emph>Hello</emph> nor <emph>Allo</emph> will be displayed since the implicit duration of the anonymous spans is 0. In contrast, the word <emph>Bonjour</emph> is contained within an anonymous span that is the child of a parallel time container (the <code>span</code> element). As such, <emph>Bonjour</emph> will be displayed since the implicit duration of the anonymous span is indefinite, resulting in the implicit duration of its <code>span</code>, <code>p</code>, <code>div</code> and <code>body</code> ancestors to also be indefinite. </p>
+<p>For the purposes of performing presentation processing, the <phrase role="strong">[construct anonymous spans]</phrase> procedure is applied before resolving begin and end times of content elements.</p>
```

Comment:
Is _all_ of [construct anonymous spans] performed before resolving begin and end times? Consider:

```
<p>
Wibble
<span begin="2s" end="3s">Wobble</span>
Wibble
</p>
```

Before 2s and after 3s, if you resolve the timing before step 2 of [construct anonymous spans] there are two contiguous anonymous spans that get replaced by a single anonymous span in step 2. Between 2s and 3s the anonymous spans are not contiguous.

If you resolve the timing after completing [construct anonymous spans] then the anonymous spans are never contiguous.

The next question is: does it ever make any difference? I think it makes no difference, but then I wonder what the point of step 2 is at all, and suspect I've missed something... Any further thoughts or ideas @palemieux @skynavga ?

Reply:
> but then I wonder what the point of step 2 is at all, and suspect I've missed something

FWIW, I think of step (2) as a clean-up pass that create a single anonymous span from the multiple text nodes that an XML element can have.